README.md

```diff
@@ -36,49 +36,25 @@ Run the application:
 $ oneview-redfish-toolkit
```

Comment:
I think we need some additional structure here.

I think we need a "getting started".   This would have things like:

Prerequisites:
Ubuntu 16.04
Python x.y
pip x.y
??? what else?

Installing the application:
(like what we have)
$ pip install git+https://github.com/HewlettPackard/oneview-redfish-toolkit.git

Run the application:
(can we show input & output for running the application) ??  example below... but use whatever is required
oneview-redfish-toolkit
>welcome to ov-rf-tk. please enter a comma separated list of OV ips you want to connect to
> 10.0.0.3,10.0.0.4
> Thank you.  your redfish service is now available at 0.0.0.0:5000


Additional information about the toolkit:
(we can add the add the additional stuff here about entries in the redfish.conf, logging, etc.)

Reply:
Done.